client/ayon_core/tools/publisher/publish_report_viewer/report_items.py

```diff
@@ -13,6 +13,11 @@ def __init__(self, plugin_data):
         self.skipped = plugin_data["skipped"]
         self.passed = plugin_data["passed"]
 
+        self.doc = plugin_data["doc"]
+        self.filepath = plugin_data["filepath"]
+        self.plugin_type = plugin_data["plugin_type"]
+        self.families = plugin_data["families"]
```

Comment:
This is also used as standalone tool where you can drop report files, but the old files won't have these new keys.
```suggestion
        self.doc = plugin_data.get("doc", "N/A")
        self.filepath = plugin_data.get("filepath", "N/A")
        self.plugin_type = plugin_data.get("plugin_type", "N/A")
        self.families = plugin_data.get("families", "N/A")
```